tests/test_fit_result.py

```diff
@@ -66,55 +66,68 @@ def test_fitting(self):
         self.assertRaises(AttributeError, getattr, *[fit_result.params, 'a__stdev'])
 
     def test_fitting_2(self):
-        np.random.seed(4242)
-        mean = (0.33, 0.28)  # x, y mean 0.3, 0.3
+        np.random.seed(43)
+        mean = (0.62, 0.71)  # x, y mean 0.7, 0.7
         cov = [
-            [0.01**2, 0.39],
-            [0.41, 0.0101**2]
+            [0.102**2, 0],
+            [0, 0.07**2]
+        ]
+        data_1 = np.random.multivariate_normal(mean, cov, 10**5)
+        mean = (0.33, 0.28)  # x, y mean 0.3, 0.3
+        cov = [  # rho = 0.25
+            [0.05 ** 2, 0.25 * 0.05 * 0.101],
+            [0.25 * 0.05 * 0.101, 0.101 ** 2]
         ]
-        data = np.random.multivariate_normal(mean, cov, 1000000)
-        mean = (0.68, 0.71)  # x, y mean 0.6, 0.4
-        cov = [[0.0102**2, 0.00], [1e-9, 0.010**2]]
-        data_2 = np.random.multivariate_normal(mean, cov, 1000000)
-        data = np.vstack((data, data_2))
+        data_2 = np.random.multivariate_normal(mean, cov, 10**5)
+        data = np.vstack((data_1, data_2))
 
         # Insert them as y,x here as np fucks up cartesian conventions.
-        ydata, xedges, yedges = np.histogram2d(data[:,1], data[:,0], bins=200, range=[[0.0, 1.0], [0.0, 1.0]])
+        ydata, xedges, yedges = np.histogram2d(data[:, 1], data[:, 0], bins=200,
+                                               range=[[0.0, 1.0], [0.0, 1.0]],
+                                               density=True)
```

Comment:
`density` should probably be `normalize` or soemthing. Not looking it up now, I want to go home.

Reply:
No the opposite, according to their docs `density` is preferred over `normed`. So why it fails now on most python versions is a mystery to me.

Reply:
Add a `--upgrade` to the travis.yml, so that `pip install --only-binary numpy,scipy,matplotlib .[all]` becomes `pip install --upgrade --only-binary numpy,scipy,matplotlib .[all]`

Reply:
doesn't seem to solve the problem